substratest/assets.py

```diff
@@ -331,6 +337,7 @@ class Testtuple(_Asset, _FutureMixin):
     certified: bool
     rank: int
     tag: str
+    metadata: typing.Dict[str, str] = None
```

Comment:
The main description says testtuples are tested, but there's still `= None` here. Could you remove it?